app/javascript/packs/globals.js

```diff
@@ -18,9 +18,23 @@ require('jquery-ui/ui/widgets/sortable');
 require('jquery-ujs');
 require('jquery.observe_field');
 require('patternfly-bootstrap-treeview');
-require('patternfly/dist/js/patternfly.min');
 require('jquery.hotkeys');
 
+// all of patternfly js except for vertical navigation (patternfly-functions-vertical-nav.js)
```

Comment:
so, this bit didn't really work,
because `require('angular-patternfly')` requires the whole thing anyway.

Fixing in https://github.com/ManageIQ/manageiq-ui-classic/pull/6449,
and removing the call to `setupVerticalNavigation` which no longer works without `patternfly-functions-vertical-nav.js`.